tests/SlimFaas.Tests/HistorySynchronizationWorkerShould.cs

```diff
@@ -39,10 +38,10 @@ public async Task SyncLastTicksBetweenDatabaseAndMemory()
         await Task.Delay(200);
```

Comment:
there's a lot of magic values that should either be centralized somewhere (a static class with const) or moved as const of the type they belong to.